pkg/csi/service/common/types.go

```diff
@@ -85,7 +85,7 @@ type CreateVolumeSpec struct {
 	// TODO: Move this StorageClassParams
 	AffineToHost            string
 	VolumeType              string
-	VsanDirectDatastoreURL  string // Datastore URL from vSan direct storage pool
+	VsanDatastoreURL        string
```

Comment:
Let's not remove the previous comment. Instead modify it to say that it is for vDPp workloads.

Reply:
makes sense.

Reply:
Done.